packages/grafana-data/src/types/navModel.ts

```diff
@@ -26,6 +26,8 @@ export interface NavLinkDTO {
   children?: NavLinkDTO[];
   highlightText?: string;
   emptyMessageId?: string;
+  // In case it is a (standalone) plugin page, the ID of the plugin which registered it.
```

Comment:
this comment is not correct, the pluginId is always present for all plugin nav nodes (from what I can tell)

Reply:
Good catch, will update the comment 👍

Reply:
Updated in #[7666dce](https://github.com/grafana/grafana/pull/57769/commits/7666dce941e685d5ba781c817249fb7ed7e839df).